File: official/docs/php/v6/batches/remove-shipments.php
<?php

$client = new \EasyPost\EasyPostClient(getenv('EASYPOST_API_KEY'));

$batch = $client->batch->retrieve('batch_...');

$batchWithoutShipments = $client->batch->removeShipments(
    $shipment->id,
    [
        'shipments' => [
            ['id' => 'shp_...']
        ]
    ]
);

echo $batchWithoutShipments;

File: official/docs/ruby/v6/batches/remove-shipments.rb
require 'easypost'

client = EasyPost::Client.new(api_key: 'EASYPOST_API_KEY')

batch = client.batch.remove_shipments(
  'batch_...',
  shipments: [
    { id: 'shp_...' },
  ],
)

puts batch
